mecab/Rakefile

```diff
@@ -32,6 +32,7 @@ class MecabPackageTask < PackagesGroongaOrgPackageTask
     [
       "almalinux-8",
       "almalinux-8-aarch64",
+      "almalinux-9",
```

Comment:
Could you remove this from this pull request?
We need to use https://repo.almalinux.org/vault/9/AppStream/Source/Packages/mecab-0.996-3.el9.3.src.rpm for AlmaLinux 9.

Reply:
I have removed.